packages/fxa-auth-server/lib/senders/emails/partials/requestDevice/en.ftl

```diff
@@ -0,0 +1,31 @@
+# "This request" refers to some action a person has taken that leads to the email.
+# Variables:
+# - $uaBrowser: the user agent's browser (e.g., Firefox Nightly)
+# - $uaOS: the user agent's operating system (e.g, MacOS)
+# - $uaOSVersion - the user agent's operating system version
+account-request-origin-device-all = This request came from { $uaBrowser } on { $uaOS } { $uaOSVersion }.
+
+# "This request" refers to some action a person has taken that leads to the email.
+# Variables:
+# - $uaBrowser: the user agent's browser (e.g., Firefox Nightly)
+# - $uaOS: the user agent's operating system (e.g, MacOS)
+account-request-origin-device-browser-os = This request came from { $uaBrowser } on { $uaOS }.
+
+# "This request" refers to some action a person has taken that leads to the email.
+# Variables:
+# - $uaBrowser: the user agent's browser (e.g., Firefox Nightly)
+account-request-origin-device-browser-only = This request came from { $uaBrowser }.
+
+# "This request" refers to some action a person has taken that leads to the email.
+# Variables:
+# - $uaOS: the user agent's operating system (e.g, MacOS)
+# - $uaOSVersion - the user agent's operating system version
+account-request-origin-device-OS-version-only = This request came from { $uaOS } { $uaOSVersion }.
```

Comment:
Any issues with the uppercase `-OS-` in the middle of this string (and L28 below)?
It seems to be lowercase `-browser-os` above on L12.